ui/components/multichain/network-list-menu/network-list-menu.tsx

```diff
@@ -559,6 +563,7 @@ export const NetworkListMenu = ({ onClose }: { onClose: () => void }) => {
       return (
         <SelectRpcUrlModal
           networkConfiguration={networkConfigurations[editedNetwork.chainId]}
+          onNetworkChange={handleNetworkChange}
```

Comment:
OK nice! So this performs the same set of actions for when you click in the network list. Yep this is valid!